src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,10 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
     public String getRandomColor() {
-        return null;
+        int index = new Random().nextInt(Color.values().length);
```

Comment:
You are using `Color.values()` twice which is not efficient since `values()` creates a new array each time it is called. Instead, you can call it once, store it in a local variable, and then use that variable to get the random color.